setup/configure_kernel_ima_module/test.sh

```diff
@@ -31,7 +31,7 @@ rlJournalStart
         # install IMA policy
         rlRun "limeInstallIMAConfig ${IMA_POLICY_FILE}"
         # clear TPM
-        if ! limeTPMEmulated; then
+        if ! limeTPMEmulated && [ -c /dev/tpmrm0 ] || [ -c /dev/tpm0 ]; then
```

Comment:
I think just a check for one of them would be enough. In tests we rely on `/dev/tpmrm0` anyway.

Reply:
Hmm that's true @kkaarreell , but do you think that `/dev/tpm0` is not used anymore? I read somewhere that it's used for older version of  TPM (1.2 or 2.0). I just think that could be useful also upgrade our tests to conditionally handle `/dev/tpm0` or `/dev/tpmrm0`.

Reply:
What I mean is that although both files are present, `/dev/tmp0' is for the direct device access while `/dev/tmprm0` it the access to TPM through a kernel resource manager and that is what we are using. In recent kernels there should always be also `/dev/tpmrm0` when `/dev/tmp0` exists. Btw, keylime won't work with TPM 1.2.

Reply:
Ok, so I'll keep just `/dev/tpmrm0`.